.eslintrc.js

```diff
@@ -19,11 +19,8 @@ module.exports = {
   },
   plugins: ["@typescript-eslint", "prettier"],
```

Comment:
Wasn't the prettier removed in the last PR?

Reply:
It was added last PR and the `prettier --check` was removed because `eslint` verifies it internally